src/main/webapp/WEB-INF/views/order/all.jsp

```diff
@@ -0,0 +1,29 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Orders</title>
+</head>
+<body>
+<h2>Orders</h2>
+<a href = "${pageContext.request.contextPath}/main"> Fruit store </a>
+<c:forEach var="order" items="${orders}">
+    <p>id = ${order.id} userId = ${order.userId}</p>
+    <table border="1">
+        <tr>
+            <th>name</th>
+        </tr>
+        <c:forEach var="product" items="${order.products}">
+            <tr>
+                <td>${product.name}</td>
+            </tr>
+        </c:forEach>
+        <a href="${pageContext.request.contextPath}/orders/delete?orderId=${order.id}">delete</a>
+    </table>
```

Comment:
make a table with 4 columns:

Order Id
User Id
Order details
Delete order

Reply:
Added jsp for admin and user, not sure about interactions for them inside project